recipes/miniscript/all/conanfile.py

```diff
@@ -24,6 +27,23 @@ class MiniscriptConan(ConanFile):
     }
     exports_sources = ["CMakeLists.txt"]
 
+    @property
+    def _min_cppstd(self):
+        return 20
```

Comment:
But the upstream says 14: https://github.com/JoeStrout/miniscript/blob/master/CMakeLists.txt#L106 🤔

Reply:
Oh no!
I will fix my mistake later.

Reply:
No problem! 😸

Reply:
@uilianries 
Sorry. I forgot my decision about required C++ standards.

While miniscript's CMakeLists.txt reuiqres C++14, miniscript source uses C++20 features. [ref](https://github.com/JoeStrout/miniscript/blob/v1.6.2/MiniScript-cpp/src/MiniScript/MiniscriptTypes.cpp#L18)

I am going to ask this mismatch on upstream.

At this point, I revert this PR to require C++20 for 1.6.2.

Reply:
Thank you @toge !!